modules/renter/hostdb/consts.go

```diff
@@ -0,0 +1,36 @@
+package hostdb
+
+import (
+	"time"
+
+	"github.com/NebulousLabs/Sia/build"
+)
+
+const (
+	defaultScanSleep = 1*time.Hour + 37*time.Minute
+	maxScanSleep     = 4 * time.Hour
+	minScanSleep     = 1*time.Hour + 20*time.Minute
+
+	maxSettingsLen = 4e3
+
+	hostRequestTimeout = 60 * time.Second
+	hostScanDeadline   = 60 * time.Second
+)
+
+var (
+	// hostCheckupQuantity specifies the number of hosts that get scanned every
+	// time there is a regular scanning operation.
+	hostCheckupQuantity = build.Select(build.Var{
+		Standard: int(250),
+		Dev:      int(4),
+		Testing:  int(5),
+	}).(int)
+
+	// scanningThreads is the number of threads that will be probing hosts for
+	// their settings and checking for reliability.
+	scanningThreads = build.Select(build.Var{
+		Standard: int(25),
+		Dev:      int(6),
+		Testing:  int(3),
+	}).(int)
```

Comment:
hmm. Definitely silly that we have 6 scanning threads for only 4 hosts. Maybe the number of scanning threads should be 3 for both? I want enough that race conditions can be spotted.

I guess ideally we'd find way to run antfarms with 20+ hosts on them.